test/extended/heapster-deployer.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
```

Comment:
Can we not just fetch the latest version of this from Github in the script?  Isn't this going to drift from what's in the other repo?